specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-03-02-preview/managedClusters.json

```
@@ -3003,9 +3003,14 @@
           "$ref": "#/definitions/AgentPoolMode"
         },
         "orchestratorVersion": {
+          "type": "string",
+          "title": "The version of Kubernetes specified by user.",
+          "description": "Both patch version <major.minor.patch> and alias minor version <major.minor> are supported. As a best practice, you should upgrade all node pools in an AKS cluster to the same Kubernetes version. The node pool version must have the same major version as the control plane. The node pool minor version must be within two minor versions of the control plane version. The node pool version cannot be greater than the control plane version. For more information see [upgrading a node pool](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#upgrade-a-node-pool)."
```

IMO I wouldn't use the phrase `alias minor version` as (AFAIK) that's language we use internally but isn't really used by the community at large.

I might say:
```suggestion
          "description": "Both patch version <major.minor.patch> and <major.minor> are supported. When <major.minor> is specified, the latest supported patch version is chosen automatically. Updating the agent pool with the same <major.minor> once it has been created will not trigger an upgrade, even if a newer patch version is available. As a best practice, you should upgrade all node pools in an AKS cluster to the same Kubernetes version. The node pool version must have the same major version as the control plane. The node pool minor version must be within two minor versions of the control plane version. The node pool version cannot be greater than the control plane version. For more information see [upgrading a node pool](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#upgrade-a-node-pool)."
```

Also, since I think the existing `kubernetesVersion` field doesn't describe this major/minor behavior, it might be good to update the docs there as well.